views/account/settings.hbs

```diff
@@ -1,261 +1,250 @@
-							<div class="input-group show_hide_password">
-								<input type="password" autocomplete="new-password" name="passwordNew" id="passwordNew"
-									data-testid="settings_password_new" class="form-control" placeholder="***************"
-									pattern="^(?=.*[A-Z])(?=.*[0-9])(?=.*[a-z])(?=.*[\-_!<>§$%&\/()=?\\;:,.#+*~']).{8,255}$"
-									title="{{> "lib/forms/password-hint" plaintext=true}}">
-								<div class="input-group-addon">
-									<a href="" title="{{$t "login.anchor.title.show_password"}}" style="cursor: pointer;"><i
-											class="fa fa-eye-slash" aria-hidden="true"></i></a>
-								</div>
-							</div>
-						</div>
-
-						<div class="form-group">
-							<label for="password_control">{{$t "account.global.label.repeatNewPassword"}}</label>
-							<div class="input-group show_hide_password">
-								<input type="password" autocomplete="new-password" name="password_control" id="password_control"
-									data-testid="settings_password_control" class="form-control" placeholder="***************"
-									title="{{> "lib/forms/password-hint" plaintext=true}}">
-								<div class="input-group-addon">
-									<a href="" title="{{$t "login.anchor.title.show_password"}}" style="cursor: pointer;"><i
-											class="fa fa-eye-slash" aria-hidden="true"></i></a>
-								</div>
-							</div>
-						</div>
-					{{/userHasPermission}}
-
-					{{> "lib/components/csrfInput"}}
-
-					<div class="form-group">
-						<button type="submit" class="btn btn-primary" data-testid="submit_new_password_btn">
-							{{$t "account.button.saveAccountSettings"}}
-						</button>
-					</div>
-				</form>
-
-				<input type="hidden" name="userId" value="{{userId}}" />
-
-				<span class="hint">
-					<span class="star-reference text-muted">*</span>
-					{{> "lib/forms/password-hint" plaintext=false style="display: inline;" class="text-muted"}}
-				</span>
+    {{#content "styles" mode="append"}}
+        <link rel="stylesheet" href="/styles/account/settings.css"/>
+    {{/content}}
+
+    {{#content "scripts" mode="append"}}
+        <script src="/scripts/settings.js" type="text/javascript" nonce="{{nonceValue}}" defer></script>
+    {{/content}}
+
+    {{#content "page"}}
+
+        {{message}}
+        {{error}}
+
+        <div class="route-account">
+            {{#unless sso}}
+            <form method="post">
+                {{#ifneq currentRole "Schüler"}}
+                <div class="form-group">
+                    <label>{{$t "global.label.firstNameColon" }}</label>
+                    <input type="text" class="form-control" name="firstName" value="{{ ../currentUser.firstName }}">
+                </div>
+
+                <div class="form-group">
+                    <label>{{$t "global.label.lastNameColon" }}</label>
+                    <input type="text" class="form-control" name="lastName" value="{{ ../currentUser.lastName }}">
```

Comment:
id tag removed here

Reply:
Please check if anything else was removed in this file. I Dont know though if these tags are really neccessary because i dont think they do anything as they are "linking" label to text input.

Reply:
done ;)